### DISCUSSION TITLE
Need Clean build

### DISCUSSION THREAD

**Discussion by Sachin9822** [ROM-builders/temporary#18336] [Q&A]
https://cirrus-ci.com/task/6007421928210432?logs=build#L2953

**Comment by Apon77** [comment]
your script even have sync issue. i could not test because of this. also i dont think its from server side. please find yourself solution

**Reply by Sachin9822** [reply]
https://github.com/ROM-builders/temporary/blob/tissot-ProjectBlaze-13-Sachin/build_rom.sh 

this is the script this doent have sync issue

**Reply by Apon77** [reply]
are you sure? checked ever?

**Reply by Lafactorial** [reply]
Same happened to me 
https://cirrus-ci.com/task/5993779484688384

**Reply by Sachin9822** [reply]
yes
